tests/test_init.py

```diff
@@ -282,3 +282,23 @@ def test_raise_invalid_threadcount():
         vmecpp.run(vmec_input, max_threads=-1)
     with pytest.raises(RuntimeError):
         vmecpp.run(vmec_input, max_threads=0)
+
+
+def test_VmecInput_validation():
```

Comment:
typically we would call this:
```suggestion
def test_vmec_input_validation():
```